app/static/js/userProfile.js

```diff
@@ -1,4 +1,21 @@
 $(document).ready(function(){
+  $("#expressInterest").on("click", function() {
+    let username = $(this).data('username')
+    let data = {"username":username}
+    $.ajax({
+        url: "/profile/"+username+"/indicateInterest",
+        type: "POST",
+        data: data,
+        success: function(s) {
+
+        },
+        error: function(request, status, error) {
+          console.log(error)
+          msgToast("Error!", "Failed to saving changes!")
```

Comment:
Correct grammar to "Failed to save changes!" or change wording since it is technically failing saving interest, changes isn't really the right word.